bitnami/mariadb/README.md

```diff
@@ -117,6 +117,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `primary.affinity`                           | Affinity for MariaDB primary pods assignment                                                                      | `{}`            |
 | `primary.nodeSelector`                       | Node labels for MariaDB primary pods assignment                                                                   | `{}`            |
 | `primary.tolerations`                        | Tolerations for MariaDB primary pods assignment                                                                   | `[]`            |
+| `primary.topologySpreadConstraints`          | Topology Spread Constraints for MariaDB primary pods assignment                                                   | `[]`            |
```

Comment:
```suggestion
| `primary.topologySpreadConstraints`          | Topology Spread Constraints for MariaDB primary pods assignment                                                   | `{}`            |
```
It should be an object instead of an array